TLM/TLM/UI/SubTools/SpeedLimits/Overlay/OverlaySegmentSpeedlimitHandle.cs

```diff
@@ -69,62 +76,58 @@ namespace TrafficManager.UI.SubTools.SpeedLimits.Overlay {
                 return;
             }
 
-            // Called for each lane in the traversed street
-            bool ForEachSegmentFun(SegmentLaneTraverser.SegmentLaneVisitData data) {
-                if (data.SegVisitData.Initial) {
-                    return true;
-                }
+            // Called for each segment in the traversed route
```

Comment:
this method does not seem to use local variables. therefore we can make it static (it can still be inside of the outer method).